crypto/fipsmodule/self_check/self_check.c

```diff
@@ -387,7 +387,11 @@ static DH *self_test_dh(void) {
   return NULL;
 }
 
-static const size_t kModuleDigestSize = SHA256_DIGEST_LENGTH;
+#if defined(OPENSSL_ANDROID)
+#define MODULE_DIGEST_SIZE SHA256_DIGEST_LENGTH
+#else
+#define MODULE_DIGEST_SIZE SHA512_DIGEST_LENGTH
+#endif
```

Comment:
all this should be removed, the integrity check was moved to `bcm.c`.

Reply:
Removed